packages/server-api/src/index.ts

```diff
@@ -11,6 +11,7 @@ export * from './mmsi/mmsi'
 export * from './propertyvalues'
 export * from './brand'
 export * from './weatherapi'
+export * from './radarapi'
```

Comment:
Instead of putting everything at the root of the server API let's use the new style adopted in history api, see below. See https://github.com/SignalK/signalk-server/pull/1653#issuecomment-3240377743

Reply:
👍